stdlib/public/core/Span/RawSpan.swift

```diff
@@ -63,13 +78,24 @@ extension RawSpan {
   ///   - buffer: an `UnsafeMutableRawBufferPointer` to initialized memory.
   ///   - owner: a binding whose lifetime must exceed that of
   ///            the newly created `RawSpan`.
+  @_disallowFeatureSuppression(NonescapableTypes)
   @_alwaysEmitIntoClient
+  @lifetime(immortal)
```

Comment:
Should be `@lifetime(buffer)`. The resulting Span depends on the validity of the buffer.